pkg/bpf/map_linux.go

```diff
@@ -735,6 +735,12 @@ func (m *Map) Dump(hash map[string][]string) error {
 	return nil
 }
 
+// DumpBatchAll returns the count of elements in the map by dumping the map
+// using batch lookup.
+func (m *Map) DumpBatchAll(nextKeyOut, keysOut, valuesOut interface{}) (int, error) {
```

Comment:
I'd expose `BatchLookup` verbatim here. `All` may be a bit redundant with `Dump`.